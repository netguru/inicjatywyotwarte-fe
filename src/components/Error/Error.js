import React from 'react'
import './Error.css'

export default function Error ({ children }) {
  return (
    <div className='error'>
      <p>{children}</p>
    </div>
  )
}
