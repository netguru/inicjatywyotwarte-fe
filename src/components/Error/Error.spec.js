import React from 'react'
import Error from './Error'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Error', () => {
  test('renders div with proper class', () => {
    const wrapper = shallow(<Error/>);
    expect(wrapper.find('div[className="error"]').length).toEqual(1);
  });

  test('renders children', () => {
    const wrapper = shallow(<Error/>);
    expect(wrapper.children.length).toEqual(1);
  });

  test('match snapshot', () => {
    const wrapper = shallow(<Error/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});


