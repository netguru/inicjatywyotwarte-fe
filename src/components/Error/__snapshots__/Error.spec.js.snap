// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error match snapshot 1`] = `
<div
  className="error"
>
  <p />
</div>
`;
